consensus/src/round_manager.rs

```diff
@@ -607,9 +643,17 @@ impl RoundManager {
     /// 1) fetch missing dependencies if required, and then
     /// 2) call process_certificates(), which will start a new round in return.
     async fn process_vote(&mut self, vote: &Vote) -> anyhow::Result<()> {
+        let round = vote.vote_data().proposed().round();
+        debug!(
+            round = round,
+            remote_peer = vote.author(),
+            event = LogEvent::ReceiveVote,
```

Comment:
Did you consider having a defined schema and using the generated setters to ensure that these fields all have the same type whenever they're logged?

Reply:
Does that mean I have to have one Schema per event type? or I'll have a giant schema with all possible fields optional?

Reply:
In order to improve maintainability I've been suggesting folks have 1 single schema that has all fields optional. Note I'm not suggesting you actually make a change just pointing out that there's other options